dm/syncer/validator_checkpoint.go

```diff
@@ -149,7 +151,7 @@ func (c *validatorPersistHelper) createTable(tctx *tcontext.Context) error {
 			revision bigint NOT NULL,
 			create_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
 			update_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-			UNIQUE KEY uk_source_schema_table_key(source, schema_name, table_name, row_pk),
+			INDEX idx_source_schema_table_key(source, schema_name, table_name, row_pk),
```

Comment:
it seems that this affects the current checkpoint, is there some extra work that needs to be done?

Reply:
as explained in the pr desc, now we use `revision` field to link checkpoint and pending data, the uk constraint need to be removed, so we can persist without using transaction to avoid "transaction too large" problem.

Reply:
If there is already a task running incremental verification, the user needs to manually delete the checkpoint and start again, right?

However, since this is an undisclosed feature, so lgtm

Reply:
checkpoint tables of validator cannot be cleaned separately, but can be removed together with task's using `start-task --remove-meta`.